spec/src/main/asciidoc/jakarta-ee.adoc

```diff
@@ -34,8 +84,72 @@ When running in an environment where Jakarta Transactions and Jakarta CDI are av
 
 When a repository method is annotated with an interceptor binding annotation, the interceptor is bound to the repository bean according to the interceptor binding annotation of the repository interface method, causing the bound interceptor to be invoked around the repository method when it runs. This enables the use of interceptors such as `jakarta.transaction.Transactional` on repository methods when running in an environment where the Jakarta EE technology that provides the interceptor is available.
 
-=== Jakarta Persistence
+=== Mapping an Entity
+
+An entity class in Jakarta Data is a regular Java class that is annotated to indicate its ability to represent objects in a database. Jakarta Data itself does not provide specific annotations for mapping entities. Instead, each Jakarta Data provider can create annotations or support existing ones, such as those used in Jakarta Persistence or Jakarta NoSQL.
```

Comment:
Let's delete this section. It duplicates/summarizes another "Mapping an Entity" section that we have above which is more detailed.  It would be confusing to have 2 separate sections both titled "Mapping an Entity".

```suggestion
```